app/src/main/java/com/dessalines/thumbkey/keyboards/CommonKeys.kt

```diff
@@ -32,6 +32,46 @@ import com.dessalines.thumbkey.utils.SlideType
 import com.dessalines.thumbkey.utils.SwipeDirection
 import com.dessalines.thumbkey.utils.SwipeNWay
 
+val abcOrNumericSwipes =
```

Comment:
Name this uppercase like the other (pseudo) constants, so `val ABC_OR_NUMERIC_SWIPES`

Reply:
Fixed! I had initially picked the camelCase name because my intention was that this constant would only be used locally (inside `CommonKeys.kt`), and not in other files. I guess it was a habit from Go to turn it into a constant starting with a lower-case letter, in that case :stuck_out_tongue: Kotlin has no way to make a `val` file-private, right? So I guess that `val`s are always package-private?

Reply:
Its no probs. Ideally we could use `const val UPPERCASE_CONST` but that isn't possible for anything but primitive types iirc.